### DISCUSSION TITLE
How to get or see the re-constructed page for a given input ?

### DISCUSSION THREAD

**Discussion by PrasannaVpk** [mindee/doctr#446] [Q&A] [answered]
I want to see the result of page-constructed image,how to get that ?

**Comment by fg-mindee** [comment, marked as answer]
Hi @PrasannaVpk :wave: 

If you mean the image synthesized from the model predictions, here is a snippet:
```python
import matplotlib.pyplot as plt
from doctr.io import DocumentFile
from doctr.models import ocr_predictor
from doctr.utils.visualization import synthetize_page

model = ocr_predictor(pretrained=True)
# PDF
doc = DocumentFile.from_pdf("path/to/your/doc.pdf").as_images()
# Analyze
result = model(doc)
# Reconstructed the first page of the document
img = synthetize_page(result.pages[0].export())
plt.imshow(img); plt.show()
```
which yields:
![synthetized_page](https://user-images.githubusercontent.com/76527547/131258531-d4b8db34-eb03-45e5-afa1-0359afac8b59.png)

Hope this helps :) 
If I misunderstood you, please specify what you meant so that we can assist you :ok_hand:

**Reply by PrasannaVpk** [reply]
Yes,this is what i expected . Thanks for the quick response.
